tests/test_oauth2client.py

```diff
@@ -209,21 +209,23 @@ def test_get_environment_gae_local(self):
 
   def test_get_environment_gce_production(self):
     os.environ['SERVER_SOFTWARE'] = ''
+    no_proxy_conf = urllib.request.build_opener(urllib.request.ProxyHandler({}))
     response = MockResponse({'Metadata-Flavor': 'Google'})
-    with mock.patch.object(urllib.request, 'urlopen',
+    with mock.patch.object(no_proxy_conf, 'open',
                            return_value=response,
-                           autospec=True) as urlopen:
+                           autospec=True) as open:
       self.assertEqual('GCE_PRODUCTION', _get_environment())
-      urlopen.assert_called_once_with(
+      no_proxy_conf.open.assert_called_once_with(
```

Comment:
i suspect you just want to use `open` here, since that's your mock. (or, you know, whatever you rename it to based on the above. `;)`)